### PR TITLE
Support creating ngram for Japanese text with quanteda package.

### PR DESCRIPTION
**Pull request by hidekoji**
# Description

Added `ngrams` argument to do_tokenize_icu.

# Checklist
Make sure you have performed following items before submitting this pull request.
If not, please describe the reason.  

- [ ] Add test cases for this fix/enhancement
- [ ] Pass devtools::check()
- [ ] Pass devtools::test()
- [ ] Test installing from github
- [x] Tested with Exploratory

### CODE REVIEW

File: R/string_operation.R [hunk 1]
```diff
@@ -195,13 +205,31 @@ do_tokenize_icu <- function(df, text_col, token = "word", keep_cols = FALSE,
...
     quanteda::tokens(what = token, remove_punct = remove_punct, remove_numbers = remove_numbers,
                      remove_symbols = remove_symbols, remove_twitter = remove_twitter,
                      remove_hyphens = remove_hyphens, remove_separators = remove_separators,
                      remove_url = remove_url) %>%
-    quanteda::tokens_wordstem() %>%
-    quanteda::dfm()
+    quanteda::tokens_wordstem()
+
+  if(ngrams > 1) { # if ngrams is greater than 1, it needs to remove stopwords before hand.
+    tokens <- quanteda::tokens_ngrams(tokens, n = c(1:ngrams))
```
Comment (hideaki) [inline R/string_operation.R:216]: c() probably is unnecessary.

File: R/string_operation.R [hunk 2]
```diff
@@ -164,6 +166,7 @@ word_to_sentiment <- function(words, lexicon="bing"){
 #' @param hiragana_word_length_to_remove Length of a Hiragana word that needs to be excluded from the result.
 #' @param summary_level Either "row" or "all". If this is "all", it ignores document and summarizes the result by token.
 #' @param sort_by Either "count" or "name"
+#' @param ngrms - by default it's 1. Pass 2 for bigram, 3 for trigram.
```
Comment (hideaki) [inline R/string_operation.R:169]: Typo: ngrms

File: R/string_operation.R [hunk 3]
```diff
@@ -195,13 +198,29 @@ do_tokenize_icu <- function(df, text_col, token = "word", keep_cols = FALSE,
...
-    quanteda::dfm()
+    quanteda::tokens_wordstem()
+
+
+  # when stopwords Language is set, use the stopwords to filter out the result.
+  if(!is.null(stopwords_lang)) {
+    stopwords_to_remove <- exploratory::get_stopwords(lang = stopwords_lang)
+    tokens <- tokens %>% quanteda::tokens_remove(stopwords_to_remove, valuetype = "fixed")
+  }
+  # For remove Japanese Hiragana handling
```
Comment (hideaki) [inline R/string_operation.R:214]: Please improve this comment.

Comment (hidekoji) [reply R/string_operation.R]: Updated comment.

File: R/string_operation.R (hunk 2 above)
Comment (hidekoji) [reply R/string_operation.R]: fixed typo.

File: R/string_operation.R (hunk 1 above)
Comment (hidekoji) [reply R/string_operation.R]: Removed the c() part.

**Pull request merged**